modules/mbedtls/crypto_mbedtls.cpp

```diff
@@ -181,6 +181,38 @@ Error X509CertificateMbedTLS::save(String p_path) {
 	return OK;
 }
 
+String X509CertificateMbedTLS::save_to_string() {
+	// Writes to a string buffer the value of the certificate.
+	String buffer;
+	mbedtls_x509_crt *crt = &cert;
+	while (crt) {
+		unsigned char w[4096];
+		size_t wrote = 0;
+		int ret = mbedtls_pem_write_buffer(PEM_BEGIN_CRT, PEM_END_CRT, cert.raw.p, cert.raw.len, w, sizeof(w), &wrote);
+		if (ret != 0 || wrote == 0) {
+			// Uh-oh. Okay nobody panic. We will just return what we have...
+			return buffer;
+		}
+
+		std::string str(w, w + wrote - 1); // Don't write the string terminator
+		buffer += str.c_str();
+		crt = crt->next;
+	}
+	return buffer;
+}
+
+Error X509CertificateMbedTLS::load_from_string(const String &p_string_key) {
```

Comment:
We need to check if the certificate is in use by a TLS session.

```suggestion
Error X509CertificateMbedTLS::load_from_string(const String &p_string_key) {
	ERR_FAIL_COND_V_MSG(locks, ERR_ALREADY_IN_USE, "Certificate is in use");
```